src/System.Data.SqlClient/tests/FunctionalTests/SqlConnectionTest.cs

```diff
@@ -30,6 +30,29 @@ public void ConnectionTest()
             Assert.Null(e);
         }
 
+
+        [Fact]
+        [PlatformSpecific(TestPlatforms.Windows)]  // Integ auth on Test server is supported on Windows right now
+        public void IntegratedAuthConnectionTest()
+        {
+            Exception e = null;
+
+            using (TestTdsServer server = TestTdsServer.StartTestServer())
+            {
+                try
+                {
+                    SqlConnectionStringBuilder builder = new SqlConnectionStringBuilder(server.ConnectionString);
+                    builder.IntegratedSecurity = true;
+                    SqlConnection connection = new SqlConnection(builder.ConnectionString);
+                    connection.Open();
+                }
+                catch (Exception ce)
+                {
+                    e = ce;
+                }
+            }
+            Assert.Null(e);
```

Comment:
This is a strange construction.  Why not just let the exception propagate rather than catching it, storing it, and then verifying nothing was stored by asserting it's null?  If the goal of the test is to verify that no exception is thrown, it's ok not to have any asserts.